.github/workflows/python-format.yml

```diff
@@ -1,6 +1,15 @@
 name: Format Python
 
-on: [push, pull_request]
+on:
+  push:
+    branches: [master, devel]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [master, devel]
+    paths-ignore:
+      - 'docs/**'
```

Comment:
I might not ignore this particular action.  I think there is python code under `docs/**` and if we don't catch formatting errors here....then we might have a failure manifest somewhere completely different once merged.

Reply:
Fixed